backend/cybnetics/endpoints.py

```diff
@@ -1,8 +1,118 @@
-from flask import Flask, jsonify
+import json
+import ast
+import jwt
+import datetime
+
+from flask import Flask, jsonify, request, session
 
 app = Flask(__name__)
 app.config.from_pyfile('./config.py')
 
-@app.route('/test', methods=['GET'])
-def test():
-    return jsonify({'hello': 'world'})
+
+db = app.config['DATABASE']
+
+# This is just a sanity check from myself
+@app.route("/")
+def get_initial_response():
+    """Welcome message for the API."""
+    # Message to the user
+    message = {
+        'apiVersion': 'v1.0',
+        'status': '200',
+        'message': 'Welcome to the Flask API'
+    }
+    # Making the message looks good
+    resp = jsonify(message)
+    # Returning the object
+    return resp
```

Comment:
this is a bit of a nit pick but you can just do `return jsonify({'message':'welcome to flask'})` to keep things short